### DISCUSSION TITLE
add a heart to pp counter if the map is loved (and another one)

### DISCUSSION THREAD

**Discussion by calebcalamity** [ppy/osu#30946] [Ideas] [answered]
**would it be possible to add a heart to pp counter if the map is loved** (beside the counter)
![osu_2024-12-02_18-36-48](https://github.com/user-attachments/assets/d221e9cd-0685-49be-ac9e-a8d8691877fa)

**Comment by peppy** [comment, marked as answer]
I don't believe this would make sense, but there may be a way to have this happen in the future using custom skin elements.
